elements/lisk-chain/src/block_reward.ts

```diff
@@ -121,3 +125,94 @@ export const calculateSupply = (
 
 	return supply;
 };
+
+export const getTotalRewardAndFees = (
+	blockInstance: BlockInstance,
+): { readonly totalFee: bigint; readonly totalBurnt: bigint } => {
```

Comment:
I would prefer to not mix the concept of `burntFee` here in this logic. Logically its just the sum of `minFee` of each transaction we burn. So all interfaces should be preferred to return `totalMinFee` and then the caller/forger/implementation would decide if that value is needed to be burnt or do what. The `totalFee` should be total fee of all transactions and the implementation can later subtract `totalFee - totalMinFee` to calculate the `burntFee` for that block. 


```suggestion
): { readonly totalFee: bigint; readonly totalMinFee: bigint } => {
```